ouroboros-consensus/src/unstable-consensus-testlib/Test/Util/Serialisation/Roundtrip.hs

```diff
@@ -106,11 +121,15 @@ roundtrip' :: forall a.
 roundtrip' enc dec a = checkRoundtripResult $ do
     let enc_a = enc a
         bs    = toLazyByteString enc_a
+        flatTerm_a = toFlatTerm enc_a
 
-    validFlatTerm (toFlatTerm enc_a)             ?! "Encoded flat term is not valid: " <> show enc_a
-    (bsRem, a' ) <- deserialiseFromBytes dec bs `onError` showByteString bs
+    validFlatTerm flatTerm_a                     ?! "Encoded flat term is not valid: " <> show enc_a
+    -- TODO: requires https://github.com/well-typed/cborg/pull/325
```

Comment:
Should this be reflected in an issue, such that we don't forget about completing this TODO once a new version of `cborg` is released?

Reply:
I added a s-r-p that contains the `cborg` fix. I can remove this comment.